contracts/src/v0.8/shared/canonical-weth-zk/WETH9ZKSync.sol

```diff
@@ -0,0 +1,87 @@
+// Submitted for verification at Etherscan.io on 2017-12-12
+
+// Copyright (C) 2015, 2016, 2017 Dapphub
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+pragma solidity ^0.8.0;
+
+// solhint-disable
+contract WETH9ZKSync {
```

Comment:
Let's add a comment on what was changed from the original version. It is clearly based on the Dapphub contract as is visible in the license above.